nsxt/nsxt_controller_details.py

```diff
@@ -1,22 +1,19 @@
 #!/usr/bin/env python
-# coding=utf-8
 #
 # Copyright © 2015 VMware, Inc. All Rights Reserved.
 #
-# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
-# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
-# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
-# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
 #
-# The above copyright notice and this permission notice shall be included in all copies or substantial portions
-# of the Software.
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
```

Comment:
This whole section for all the files should just be:

```
# Copyright © 2019 VMware, Inc. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0
```

Reply:
Change has been implemented.